src/logging_layer.rs

```diff
@@ -0,0 +1,14 @@
+extern crate log;
+use env_logger::Builder;
+
+pub fn initialize_logging_layer() {
+    let mut builder = Builder::new();
+    let log_level = std::env::var("LOG_LEVEL").unwrap_or_else(|_| "info".to_string());
+    match log_level.to_lowercase().as_str() {
```

Comment:
might as well add the other levels (trace, debug, off)

Reply:
Added with [1c399ce91b204116f4644f121e2f9f8c3bda81c6](url)